pym/portage/dbapi/vartree.py

```diff
@@ -3851,6 +3881,17 @@ def eerror(lines):
 		phase.start()
 		phase.wait()
 
+		try:
+			print('!!!\nRead INSTALL_MASK from %s\n!!!' % os.path.join(inforoot, "INSTALL_MASK"))
+			with io.open(_unicode_encode(
+						os.path.join(inforoot, "INSTALL_MASK"),
+						encoding=_encodings['fs'], errors='strict'),
+					mode='r', encoding=_encodings['repo.content'],
+					errors='replace') as f:
+				install_mask = f.read().split()
+		except EnvironmentError as e:
+			install_mask = self.settings.get('INSTALL_MASK', '').split()
```

Comment:
This INSTALL_MASK fallback means that a new INSTALL_MASK setting can create orphan files during unmerge!

Reply:
maybe create an placeholder build-info/INSTALL_MASK file so when know when the fallback is not needed, like we do for DEFINED_PHASES

Reply:
Done. Also removed that debug print above.